instrumentation/opentelemetry-instrumentation-pika/src/opentelemetry/instrumentation/pika/utils.py

```diff
@@ -27,11 +30,18 @@ def keys(self, carrier: CarrierT) -> List[str]:
 
 _pika_getter = _PikaGetter()
 
+HookT = Callable[[Span, bytes, BasicProperties], None]
```

Comment:
Please use `TypeVar`

Reply:
I am not sure `TypeVar` is the right thing to use here

I am also trying to be consistent with other libraries in this project (see [urllib3](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py#L92) for example)

Reply:
I think that TypeVar is the right solution here, but I do see that `urlib3`, `asgi` and `starlette` instrumentations type like so.